client/post-editor/editor-ground-control/index.jsx

```diff
@@ -26,7 +26,7 @@ const Card = require( 'components/card' ),
 import { setDate } from 'state/ui/editor/post/actions';
 
 function isPostEmpty( props ) {
-	return ( props.isNew && ! props.isDirty ) || ! props.hasContent;
+	return ! props.hasContent || ( ! props.isNew && ! props.isDirty );
```

Comment:
I'm guessing the original logic was such that if the post is new and hasn't been modified, it must be empty? Do we even need to check this? Can't we test only against `hasContent` ?